lib/ansible/modules/windows/win_disk_management.py

```diff
@@ -0,0 +1,560 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2017, Marc Tschapek <marc.tschapek@itelligencegroup.com>
+#
+# This file is part of Ansible
```

Comment:
This can be simplified like other documentation modules now.

Reply:
replaced